### ISSUE TITLE
[Question] Aren't your normalization functions doing standardization?

### ISSUE THREAD

**Issue by r-y1** [PKU-MARL/omnisafe#171]
### Required prerequisites

- [X] I have read the documentation <https://omnisafe.readthedocs.io>.
- [X] I have searched the [Issue Tracker](https://github.com/PKU-MARL/omnisafe/issues) and [Discussions](https://github.com/PKU-MARL/omnisafe/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [X] Consider asking first in a [Discussion](https://github.com/PKU-MARL/omnisafe/discussions/new).

### Questions

I mean, this is normalization:
![image](https://user-images.githubusercontent.com/81631133/226784672-458dccd5-6f49-4937-b7b2-52d0ec509d7d.png)
and, this is standardization:
![image](https://user-images.githubusercontent.com/81631133/226784760-6830df59-90eb-4fed-bae6-3c7b3f441c07.png)

this is an example of your code:
```python
    def normalize(self, data: torch.Tensor) -> torch.Tensor:
        """Normalize the _data."""
        data = data.to(self._mean.device)
        self._push(data)
        if self._count <= 1:
            return data
        output = (data - self._mean) / self._std
        return torch.clamp(output, -self._clip, self._clip)
```

**Comment by Gaiejj** [comment]
Sure, this name style may not perfect. Omnisafe currently is under heavy development. We will polish all function, class and variable name in future version. Thanks for your advice !

**Comment by rockmagma02** [comment]
We apologize for any confusion, but we don't believe that it's a mistake or typo. 

Refer to Wikipedia [Normalization (statistics)](https://en.wikipedia.org/wiki/Normalization_(statistics)), the [Standard score](https://en.wikipedia.org/wiki/Standard_score), or 'standardization' you said is one of the normalization techs. And refer other RL library, like [spinning up](https://github.com/openai/spinningup) also use 'normalization' to refer $(x - \mathbb{E}x)/\sigma(x)$, for example in line 79 in [PPO](https://github.com/openai/spinningup/blob/master/spinup/algos/pytorch/ppo/ppo.py). And the same tools used in [gymnasiums](https://github.com/Farama-Foundation/Gymnasium/tree/main), (`NormalizeObservation`, `NormalizeReward`) also use $(x - \mathbb{E}x)/\sigma(x)$ as their normalization tech, you check it on line 95 in [normaize.py](https://github.com/Farama-Foundation/Gymnasium/blob/main/gymnasium/wrappers/normalize.py).

Thank you for your interest in our library and suggestions. Please let us know if you have any further questions or concerns.
